### ISSUE TITLE
How to fit data in the MarkovModel?

### ISSUE THREAD

**Issue by Maryia-M** [pgmpy/pgmpy#835]
In the book "Mastering Probabilistic Graphical Models using Python" fit operation for MarkovModel used the same as for BayesianModel. However when I try to use it I get "AttributeError: 'MarkovModel' object has no attribute 'fit'" and there is not defenition for 'fit' in MarkovModel.py. So how can I fit data in MarkovModel?

**Comment by mithun0602** [comment]
Is there any update on this. I did search the MarkovModel.py and it seems there is no fit method. Neither fit method available in MarkovChain.py. So how to fit model with data

**Comment by ankurankan** [comment]
@mithun0602 Unfortunately this is not implemented yet, and Markov model is not a priority for us at the moment, so I don't have a timeline for this.

**Comment by Ozziko** [comment]
@ankurankan thanks for pgmpy. Is there any update on MRF fitting/learning? If not, can you suggest a good alternative to people who work on MRFs?
Thanks!

**Comment by ankurankan** [comment]
@Ozziko No, we still do not have a timeline for this in pgmpy. I do not know of any python packages which implement generic MRFs unless you need some special cases like HMM (scikit-learn has an implementation of that). But I do think there are some R packages that might be helpful.

**Comment by Ozziko** [comment]
@ankurankan ok, many thanks!

**Comment by teng-gao** [comment]
Hey authors, thanks very much for the package. Is it still not possible perform inference on MRFs? I see that the tutorial for MarkovModel is not complete yet.

https://pgmpy.org/detailed_notebooks/4.%20Markov%20Models.html
